sys/aarch64/boot.c

```diff
@@ -112,113 +127,96 @@ __boot_text static void drop_to_el1(void) {
   }
 
 el1_entry:
+  WRITE_SPECIALREG(SPSR_EL1, PSR_DAIF);
   return;
 }
 
-#define AARCH64_PHYSADDR(x) ((paddr_t)(x) & (~KERNEL_SPACE_BEGIN))
-
-__boot_text static void clear_bss(void) {
-  uint64_t *ptr = (uint64_t *)AARCH64_PHYSADDR(__bss);
-  uint64_t *end = (uint64_t *)AARCH64_PHYSADDR(__ebss);
-  while (ptr < end)
-    *ptr++ = 0;
-}
-
-/* Create direct map of whole physical memory located at DMAP_BASE virtual
- * address. We will use this mapping later in pmap module. */
+#define PHYSADDR(x) ((paddr_t)(x) & (~KERNEL_SPACE_BEGIN))
+#define VIRTADDR(x) ((vaddr_t)(x) | KERNEL_SPACE_BEGIN)
 
-__boot_text static paddr_t build_page_table(void) {
+__boot_text static pde_t *early_pde_ptr(pde_t *pde, int lvl, vaddr_t va) {
```

Comment:
Sometimes we use `__boot_text static` and sometimes `static __boot_text`. As most of the functions utilize the former, please fix the other functions.

Reply:
Done!